website/docs/packages/reference/package.md

```diff
@@ -1,6 +1,9 @@
-# Package Settings
+# Module
+
+Modules in Appsmith are integral components of a Package tailored. They are reusable components containing queries and JSObjects that can yield results across multiple applications within the same workspace.
```

Comment:
Not sure if having this written in this way couldn't lead to some wrong expectation that a module can contain more than 1 query of JS Object